packages/frontend/app/(dashboard)/course/[cid]/async_centre/components/AsyncQuestionCard.tsx

```diff
@@ -159,7 +161,7 @@ const AsyncQuestionCard: React.FC<AsyncQuestionCardProps> = ({
             <div className="mb-1 flex justify-between">
               <div className="flex flex-grow">
                 {showUser && (
-                  <>
+                  <React.Fragment>
```

Comment:
why are you changing this shorthand syntax for a react fragment into the longhand one? (doing `<>` and `<React.Fragment>` is the same thing https://react.dev/reference/react/Fragment https://www.reddit.com/r/reactjs/comments/9yy3f4/or_reactfragment/

Reply:
I'm against shorthands for react.fragment, but if it's the norm for this project I'll give in :(